language/naturalapi.go

```diff
@@ -0,0 +1,85 @@
+package language
+
+import (
+	"context"
+	"notes-service/models"
+	"os"
+
+	glang "cloud.google.com/go/language/apiv1"
+	"cloud.google.com/go/language/apiv1/languagepb"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var protobufEnumToKeywordType = map[languagepb.Entity_Type]models.KeywordType{
+	languagepb.Entity_UNKNOWN:       models.Unknown,
+	languagepb.Entity_PERSON:        models.Person,
+	languagepb.Entity_LOCATION:      models.Location,
+	languagepb.Entity_ORGANIZATION:  models.Organization,
+	languagepb.Entity_EVENT:         models.Event,
+	languagepb.Entity_WORK_OF_ART:   models.WorkOfArt,
+	languagepb.Entity_CONSUMER_GOOD: models.ConsumerGood,
+	languagepb.Entity_OTHER:         models.Other,
+	languagepb.Entity_PHONE_NUMBER:  models.PhoneNumber,
+	languagepb.Entity_ADDRESS:       models.Address,
+	languagepb.Entity_DATE:          models.Date,
+	languagepb.Entity_NUMBER:        models.Number,
+	languagepb.Entity_PRICE:         models.Price,
+}
+
+type NaturalAPIService struct {
+	Service
+	client *glang.Client
+}
+
+func (s *NaturalAPIService) Init() error {
+	jsonCredential := os.Getenv("JSON_GOOGLE_CREDS")
```

Comment:
Let's maybe find a better name for this environment variable +/ use the `kingpin.App` in the main function to initialise it and manage it. Let's not forget to add it to the README of the service as well.